### PR TITLE
Add Test Case for RoomNumber

### PR DESCRIPTION
**Pull request by nyankoclaws**

### CODE REVIEW

**Review by PallonCX — APPROVED** [review]
LGTM

File: src/test/java/seedu/address/model/person/RoomNumberTest.java [hunk 1]
```diff
@@ -36,6 +37,28 @@ public void isValidRoomNumber() {
...
+        assertFalse(RoomNumber.isValidRoomNumberWDate("sw-12-12"));
+
+        LocalDate today = LocalDate.now();
+        LocalDate yesterday = today.minusDays(1);
+        LocalDate tomorrow = today.plusDays(1);
+
+        // valid addresses & dates
+        assertTrue(RoomNumber.isValidRoomNumberWDate("sw-12-12 " + today.toString()));
+        assertTrue(RoomNumber.isValidRoomNumberWDate("kw-01-02 " + tomorrow.toString()));
+        assertTrue(RoomNumber.isValidRoomNumberWDate("kms-03-01 " + yesterday.toString()));
```
Comment (JustWeiHao) [inline src/test/java/seedu/address/model/person/RoomNumberTest.java:53]: Hmm, so all three cases are expected to be `True`?

Comment (PallonCX) [reply src/test/java/seedu/address/model/person/RoomNumberTest.java:53]: Owh yea, the last modified date shouldn't be later than today right?

Comment (nyankoclaws) [reply src/test/java/seedu/address/model/person/RoomNumberTest.java:53]: Initial intention was to allow all dates in the correct format and check later on with isValidDate as it's easier to do in the constructor itself. But I could do either... So lmk if I should change it?

**Review by JustWeiHao — APPROVED** [review]
LGTM

**Review by donwong2308 — APPROVED** [review]
LGTM

**Review by larainezo — APPROVED** [review]
LGTM

**Pull request merged**